regression-tests/runalldirs

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-set -e
```

Comment:
A hidden change in this PR, regression tests will now continue until all tests are completed. This is better for getting feedback on exactly which tests failed.